packages/shared/package.json

```diff
@@ -2,9 +2,27 @@
   "name": "@clerk/shared",
   "version": "1.0.0",
   "description": "Internal package utils used by the Clerk SDKs",
-  "sideEffects": false,
   "files": [
-    "dist"
+    "dist",
+    "browser",
+    "callWithRetry",
+    "color",
+    "cookie",
+    "date",
+    "deprecated",
+    "error",
+    "file",
+    "globs",
+    "handleValueOrFn",
+    "isomorphicAtob",
+    "keys",
+    "loadScript",
+    "localStorageBroadcastChannel",
+    "poller",
+    "proxy",
+    "underscore",
+    "url",
+    "react"
```

Comment:
❓ @LekoArts why are these added at top level ?

Reply:
@LekoArts I was wondering the same thing as I was running into `turbo clean` issues. 

The npm script for shared only runs `rimraf` on `./dist`.

Are we able to nest these?

Thanks!

Reply:
Additionally, we'll need to add any directories/etc into `turbo.json` as, right now, it only lists `"dist/**"` as an output.

Reply:
@panteliselef Please see https://github.com/clerkinc/javascript/pull/1950